myskoda/models/info.py

```diff
@@ -189,7 +189,7 @@ class Info(DataClassORJSONMixin):
     name: str
     capabilities: Capabilities
     device_platform: str = field(metadata=field_options(alias="devicePlatform"))
-    service_partner: ServicePartner = field(metadata=field_options(alias="servicePartner"))
+    service_partner: ServicePartner | None = field(default=None, metadata=field_options(alias="servicePartner"))
     workshop_mode_enabled: bool = field(metadata=field_options(alias="workshopModeEnabled"))
```

Comment:
The linter trips over line lengths here and there's also some requirement about default fields coming after all non-default fields. Can you change it to this:

```suggestion
    workshop_mode_enabled: bool = field(metadata=field_options(alias="workshopModeEnabled"))
    service_partner: ServicePartner | None = field(
        default=None, metadata=field_options(alias="servicePartner")
    )
```

tip: auto-formatting with ruff (https://docs.astral.sh/ruff/) handles this.

Reply:
Fixed the ordering of the fields